pages/item/_.vue

```diff
@@ -339,9 +343,11 @@
         });
     },
 
-    mounted() {
+    created() {
       this.cardGridClass = this.isRichMedia && 'card-grid-richmedia';
```

Comment:
isRichMedia should be updated in the plugin to also check not just HTML video, but also any Playable Media in the case when the mediaPlayer is enabled.

Reply:
Addressed in 9031114